pkgs/development/python-modules/robotframework-seleniumlibrary/default.nix

```diff
@@ -0,0 +1,22 @@
+{ stdenv, buildPythonPackage, fetchPypi, robotframework, selenium }:
+
+buildPythonPackage rec {
+  version = "3.2.0";
+  pname = "robotframework-seleniumlibrary";
+
+  src = fetchPypi {
+    inherit pname version;
+    sha256 = "113ee01aa4fbfd322d4816e5704c0187ffcf6a047ea012a66d305f250ab485db";
+  };
+
+  # no tests included in PyPI tarball
```

Comment:
Should we instead fetch from GitHub, then?

On GitHub for `robotframework/SeleniumLibrary` (other than for `Selenium2Library`), these two directories are included:
- [`atest`](https://github.com/robotframework/SeleniumLibrary/tree/v3.2.0/atest)
- [`utest`](https://github.com/robotframework/SeleniumLibrary/tree/v3.2.0/utest)

Reply:
sadly the Acceptance Tests (atest directory) require chrome headless. I've updated with checking the Unit Tests though